redirect/README.md

```diff
@@ -0,0 +1,33 @@
+
+# Network Setup
+
+## VirtualBox
+
+- Host-only Network
+  - vboxnet0, 192.168.56.1/24, dhcpd=enable
+- Internal Network
+  - intnet (10.0.5.0/24)
+  - fwnet (10.0.6.0/24)
+
+## Gateway
+
+- Hostname: gw
+- Network
+  - eth0: (bridge to wifi)
+  - eth1: 10.0.5.1/24 08:00:27:b0:90:8f (intnet)
+  - eth2: 10.0.6.1/24 08:00:27:ca:8c:88 (fwnet)
```

Comment:
LAN/WAN?

Reply:
you mean intnet and fwnet? they are both LAN's, the WAN is the eth0 network, which will vary for each setup.